fares-data-build-tool/fdbt-site
# feat: review and remove from selected operators

# Description

-   This PR introduces the review and remove functionality to allow a user to remove any operators that they have previously searched for and then selected. 

# Testing instructions

-   Pull down this branch and run the site locally. Validate that the page behaves as expected in terms of all three of its components: (i) Search (ii) Add operators and (iii) Remove Operators.
-   Also validate that the page errors in the right way for each of the three components of functionality.

# Type of change


# Checklist:
